pkg/batchfetcher/config.go

```diff
@@ -0,0 +1,18 @@
+package batchfetcher
+
+import t "github.com/filecoin-project/mir/pkg/types"
+
+// ModuleConfig determines the IDs of the modules the batch fetcher interacts with.
+type ModuleConfig struct {
+	Self         t.ModuleID // Own ID.
+	Availability t.ModuleID // ID of the factory module containing the availability modules.
+	Destination  t.ModuleID // ID of the module to deliver the produced event stream to (usually the application).
+}
+
+func DefaultModuleConfig() *ModuleConfig {
+	return &ModuleConfig{
+		Self:         "batchFetcher",
```

Comment:
Do we want to use snakeCase for multi-word module IDs? Another option would be to use the go package naming convention (withnoclearseparators).

Reply:
No strong feelings about this. gopackagenamingisfinewithme :)

Reply:
No strong feelings on my side either :)
`batchdb` will probably need to be changed to `batchDB` if we adopt the snake case. Other module IDs were single words so far, as far as I can see.

Reply:
Fixed.